tartufo/scanner.py

```diff
@@ -150,7 +152,7 @@ class ScannerBase(abc.ABC):  # pylint: disable=too-many-instance-attributes
     global_options: types.GlobalOptions
     logger: logging.Logger
     _scan_lock: threading.Lock = threading.Lock()
-    _excluded_findings: Tuple[str, ...] = ()
+    _excluded_signatures: Union[Tuple[Dict[str, str], ...], Tuple[str, ...]] = ()
```

Comment:
This should still only ever be `Tuple[str, ...]`, right? This is never the value directly from `global_options`, but only the eventually interpreted value, which would always be a tuple of strings.
```suggestion
    _excluded_signatures: Tuple[str, ...] = ()
```